UPGRADING_TO_2.0.md

```diff
@@ -32,7 +32,6 @@ assists users migrating to a new version.
 - [Step 3: Upgrade Airflow DAGs](#step-3-upgrade-airflow-dags)
   - [Change to undefined variable handling in templates](#change-to-undefined-variable-handling-in-templates)
   - [Changes to the KubernetesPodOperator](#changes-to-the-kubernetespodoperator)
-- [Step 4: Update system configurations](#step-4-update-system-configurations)
```

Comment:
If this is removed, the following "Step 5" should be updated to "4"

Reply:
Just notice there is another existing section name mistake: there are two "Step 3". Would be great if it can be fixed as well

Reply:
done in 839ad45